docker/orthanc/orthanc.json

```diff
@@ -14,16 +14,24 @@
     "ConnectionRetryInterval": 5,
     "IndexConnectionsCount": 2
   },
-  "Dicom-DateTruncation": {
-    "dateformat": ["YYYY","04","10"]
-  },
-  "Dicom-Filter": {
-    "blacklist": [
-      "0010"
-    ],
-    "whitelist": [
-      "0010,0030"
-    ]
+  "DataAnon": {
+    "Hardlinks": {
+      "by-study-date": "0008,0020",
+      "by-pid": "0010,0020",
+      "by-dob": "0010,0030"
+    },
+    "DateTruncation": {
```

Comment:
Do we have documentation of the date truncation format?

Reply:
No we need documentation for all of the json formatting. #67